src/components/HomeLandingPage/HomeLandingPage.tsx

```diff
@@ -40,24 +43,71 @@ const HomeLandingPage = () => {
   useAllEvents();
 
   // Fetch events from cache
-  const { data } = useQuery(GET_ALL_EVENTS);
+  const { data } = useQuery(FETCH_ALL_EVENTS);
```

Comment:
Because you're using `FETCH_ALL_EVENTS` here instead of `GET_ALL_EVENTS`, we no longer see the newly added event without refreshing the page. (Since in `EventCreationPage.tsx`, we only write to the cache for the `GET_ALL_EVENTS` query.

Reply:
If using `GET_ALL_EVENTS` is causing any issues, I'm just going to say let's take out the `@client` and add all the fields in `FETCH_ALL_EVENTS` to the existing `GET_ALL_EVENTS` query. @petorsong thoughts?

Reply:
since we don't understand what `@client` does right now, i'm fine with just having 1 query - it's currently too confusing to maintain both

Reply:
yeah I'm just going to get rid of `FETCH_ALL_EVENTS` and change it to the `GET_ALL_EVENTS` query.